cli/install/install.go

```diff
@@ -909,23 +997,31 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 			logFile)
 	} else {
 		log.Infof("Downloading tarantool...")
-		err = downloadRepo(search.GitRepoTarantool, tarVersion, path, logFile, installCtx.verbose)
+		if versionFound {
+			err = downloadRepo(search.GitRepoTarantool, tarVersion, path, logFile,
+				installCtx.verbose)
+		} else {
+			err = downloadRepo(search.GitRepoTarantool, "master", path, logFile, installCtx.verbose)
+			util.ExecuteCommand("git", installCtx.verbose, logFile, path, "checkout", tarVersion)
```

Comment:
But this is under !versionFound condition. What is the value of the `tarVersion` here?

Reply:
tarVersion in this case is a hash